content/guides/10-minute-tutorial.md

```diff
@@ -293,6 +293,13 @@ class RunCucumberTest
 {{% block "javascript" %}}
 We'll start by creating a new directory and an empty Node.js project.
 
+Sometime, before that we need to install cucumber globally by running this command:
```

Comment:
Can we alter this to hint "why" this is required. i.e. possibly because you are on Linux.

Reply:
If this is linux specific, I'd prefer to add it inside a "Note" block. (For example, see the "Example mapping" block)

Reply:
Thank you for pointing this out and adding to the docs. Would you be able to make the requested change? Let us know if you need help.